taxi/forms.py

```diff
@@ -0,0 +1,62 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def licence_validation(license_num):
+    licence_num_len = 8
+    # decided to to all-in-one check so a user will see all the requirements in
+    # case of any validation error
+    if (
+            len(license_num) != licence_num_len
+            or not (license_num[:3].isupper() and license_num[:3].isalpha())
+            or not license_num[4:].isdigit()
+    ):
```

Comment:
It would be better to write 
`if not (here conditions that should be fulfilled joined by 'and') `

Reply:
👍